packages/platforms/react-native/lib/native.ts

```diff
@@ -7,8 +7,32 @@ declare const global: {
 
 const isTurboModuleEnabled = () => global.__turboModuleProxy != null
 
-export const NativeBugsnagPerformance = isTurboModuleEnabled()
+const LINKING_ERROR =
+  'The package \'BugsnagReactNativePerformance\' doesn\'t seem to be linked.'
+
+export const NativeBsgModule = isTurboModuleEnabled()
   ? TurboModuleRegistry.get('BugsnagReactNativePerformance')
   : NativeModules.BugsnagReactNativePerformance
 
-export default NativeBugsnagPerformance as Spec | null
+const NativeBugsnagPerformance = NativeBsgModule || new Proxy(
+  {
+    getDeviceInfo: () => undefined,
+    requestEntropy: () => '',
+    requestEntropyAsync: async () => '',
+    getNativeConstants: () => ({ CacheDir: '', DocumentDir: '' }),
+    exists: async (path: string) => false,
+    isDir: async (path: string) => false,
+    ls: async (path: string) => [],
+    mkdir: async (path: string) => '',
+    readFile: async (path: string, encoding: string) => '',
+    unlink: async (path: string) => { },
+    writeFile: async (path: string, data: string, encoding: string) => { }
+  },
+  {
+    get () {
+      throw new Error(LINKING_ERROR)
+    }
+  }
+)
```

Comment:
At the moment this will never actually use the fallback implementation and the proxy will just throw an error if any of the methods are called (that's what the get handler does)

I don't think we need to use a proxy here - we can just return the object directly if the native module is null. 

```suggestion
const NativeBugsnagPerformance = NativeBsgModule || {
  getDeviceInfo: () => undefined,
  requestEntropy: () => '',
  requestEntropyAsync: async () => '',
  getNativeConstants: () => ({ CacheDir: '', DocumentDir: '' }),
  exists: async (path: string) => false,
  isDir: async (path: string) => false,
  ls: async (path: string) => [],
  mkdir: async (path: string) => '',
  readFile: async (path: string, encoding: string) => '',
  unlink: async (path: string) => { },
  writeFile: async (path: string, data: string, encoding: string) => { }
}
```

(you might also need to update the Spec interface in `NativeBugsnagPerformance.ts` because `getDeviceInfo` might return undefined)

Reply:
I think it would be good to have a unit test for this as well